config/runtime.exs

```diff
@@ -8,7 +8,8 @@ config :skate,
   restrict_environment_access?: System.get_env("RESTRICT_ENVIRONMENT_ACCESS") == "true",
   base_tileset_url: System.get_env("BASE_TILESET_URL"),
   satellite_tileset_url: System.get_env("SATELLITE_TILESET_URL"),
-  aws_place_index: System.get_env("AWS_PLACE_INDEX")
+  aws_place_index: System.get_env("AWS_PLACE_INDEX"),
+  environment_name: System.get_env("SENTRY_ENV", "missing-env")
```

Comment:
Can we use [`ENVIRONMENT_NAME`](https://github.com/mbta/devops/blob/master/terraform/modules/app-skate-ecs/main.tf#L18C15-L18C31)?

Reply:
Yes, but just out of curiosity, why is that preferred? Just so that we're not sort of "leaking" sentry into non-sentry parts of the app?

Reply:
I don't really think of Sentry as the authoritative source, so I'd personally rather keep sentry things sentry things, and have environment be the source.

Reply:
I did mark this as a nit and non-blocking though, sorry I didn't do so sooner.

Reply:
Ah it's all good - I was curious about your reasoning, but it was easy enough to do (and it is done now 😄)